info/v2/container.go

```diff
@@ -52,6 +52,14 @@ type MemorySpec struct {
 	SwapLimit uint64 `json:"swap_limit,omitempty"`
 }
 
+type ContainerInfo struct {
+	// Describes the container.
+	Spec ContainerSpec `json:"spec,omitempty"`
```

Comment:
What do you mean by "essentially breaking the existing API"? I'm trying to change the minimum in this PR since it's blocking the stats API implementation, and also since I think you've already done the necessary work for this in https://github.com/google/cadvisor/pull/1035? Is there a reason not to wait for https://github.com/google/cadvisor/pull/1035 to remove the pointer fields?